modules/@apostrophecms/admin-bar/ui/apos/components/TheAposAdminBar.vue

```diff
@@ -3,7 +3,7 @@
     <div class="apos-admin-bar-spacer" ref="spacer" />
     <nav class="apos-admin-bar" ref="adminBar">
       <div class="apos-admin-bar__row">
-        <AposLogo class="apos-admin-bar__logo" />
+        <AposAdminBarLogo class="apos-admin-bar__logo" />
```

Comment:
Uses a separate logo, because the invisible tile creates unwanted alignment issues

Reply:
We should avoid naming things based on their context, as opposed to what they are (e.g., `AposAdminBarDark`). You can change this if you want, but I also don't want to hold this up more than necessary.